src/ray/gcs/test/redis_async_context_test.cc

```diff
@@ -74,22 +71,22 @@ TEST_F(RedisAsioTest, TestRedisCommands) {
   ASSERT_TRUE(shard_context
                   ->Connect(std::string("127.0.0.1"),
                             TEST_REDIS_SERVER_PORTS.front(),
-                            /*password=*/std::string())
+                            /*password=*/
+                            std::string())
                   .ok());
 
   io_service.run();
 }
-
 }  // namespace gcs
-
 }  // namespace ray
 
 int main(int argc, char **argv) {
   InitShutdownRAII ray_log_shutdown_raii(ray::RayLog::StartRayLog,
                                          ray::RayLog::ShutDownRayLog,
                                          argv[0],
                                          ray::RayLogLevel::INFO,
-                                         /*log_dir=*/"");
+                                         /*log_dir=*/
```

Comment:
Doesn't make sense to me?

Reply:
Well, this is copied from other tests, so maybe the linter is mis-configured? Because CI passed.

Reply:
Linter usually has multiple acceptable formats, I'm just curious how these changes are made, and think it's not necessary

Reply:
fixed